### PR TITLE
Fix the timestamp for for the events so that they show up in the present time (they did far in the past)

### PR DESCRIPTION
**Pull request by rht**
@fxfactorial the commits for the timestamp fix accidentally got squashed (during a `git rebase -i`) into the last one. In short, the err was that I use second for the unixtime, while the frontend assumes it to be in millisecond.

Address https://github.com/fxfactorial/silicondzor/issues/6#issuecomment-272078221

### CODE REVIEW

**Comment by rht** [comment]
Commits untangled, also tested in frontend.

File: backend/server.js [hunk 1]
```diff
@@ -64,6 +64,23 @@ let register_email_users = {};
...
+    client_id: process.env.ITERATE_FB_APP_ID,
+    client_secret: process.env.ITERATE_FB_APP_SECRET,
+    grant_type: 'client_credentials'
+  }, res => {
+    if (!res || res.error) {
+      console.log(!res ? 'error occurred during the token request' : res.error);
+      return;
+    }
+    const { access_token } = res;
+    FB.setAccessToken(access_token);
```
Comment (rht) [inline backend/server.js:79]: This may not be necessary since the `FB` is immutable.

File: backend/server.js [hunk 2]
```diff
@@ -72,50 +89,36 @@ const groups = { iterate: 410797219090898,
...
-  $start: Math.floor(Date.parse(each.start_time)/1000),
-  $end: each.end_time ? Math.floor(Date.parse(each.end_time)/1000) : 0,
-  $description: each.description,
-  $creator: group_name,
-  $url: `https://facebook.com/events/${each.id}`,
-  $id: `fb-${each.id}`
-});
-	});
+      res.data.forEach(each => {
+        const start = (new Date(each.start_time)).getTime();
```
Comment (rht) [inline backend/server.js:105]: const safety in js ;)

Comment (rht) [reply backend/server.js:105]: TIL

File: backend/server.js (hunk 1 above)
Comment (rht) [reply backend/server.js:79]: ic, it only prevents rebinding. `Object.freeze` is for immutability instead.

File: backend/server.js (hunk 2 above)
Comment (fxfactorial) [reply backend/server.js:105]: You might as well but FYI const has a little bit of an overhead compared to var.

File: backend/server.js (hunk 1 above)
Comment (fxfactorial) [reply backend/server.js:79]: Also semantically would be better to use `console.error`, not `console.log`. In any case there's a ticket to use `morgan`, apparently someone is working on that.

**Comment by fxfactorial** [comment]
@rht Awesome thanks! Will deploy now

**Pull request merged**